code/game/machinery/computer/arcade/arcade_game.dm

```diff
@@ -193,7 +193,7 @@
 				new /obj/item/device/maracas/cubanpete(holder.loc)
 				message_admins("[key_name_admin(usr)] has outbombed Cuban Pete and been awarded explosive maracas.")
 				log_game("[key_name_admin(usr)] has outbombed Cuban Pete and been awarded explosive maracas.")
-				holder.New()
+				holder.update_game()
```

Comment:
Dont necessarily want it to qdel the old game, just reset it.

Could want it to hold high scores

Reply:
I assumed that was the intent of the old version; am I reading that wrong or was the old version just overwriting the datum anyway?
I'll change but I want to make sure

Reply:
Oh damn, that does seem to be the intent. That sucks